packages/runtime-core/src/componentPublicInstance.ts

```diff
@@ -455,8 +455,8 @@ export const PublicInstanceProxyHandlers: ProxyHandler<any> = {
     descriptor: PropertyDescriptor
   ) {
     if (descriptor.get != null) {
-      this.set!(target, key, descriptor.get(), null)
-    } else if (descriptor.value != null) {
+      target.$.accessCache[key] = 0;
```

Comment:
nit: maybe add a comment here to explain that this invalidates the cache key to avoid evaluating the getter